extensions/iam/decentralized-identity/dummy-credentials-verifier/src/main/java/org/eclipse/dataspaceconnector/iam/verifier/DummyCredentialsVerifier.java

```diff
@@ -37,7 +39,9 @@ public DummyCredentialsVerifier(Monitor monitor) {
     }
 
     @Override
-    public Result<Map<String, Object>> verifyCredentials(String hubBaseUrl, PublicKeyWrapper othersPublicKey) {
+    public Result<Map<String, Object>> verifyCredentials(DidDocument didDocument) {
+
+        var hubBaseUrl = didDocument.getService().stream().filter(service -> service.getType().equals(DidConstants.HUB_URL)).map(Service::getServiceEndpoint).findFirst().orElseThrow();
```

Comment:
nit: just for readability.
```suggestion
        var hubBaseUrl = didDocument.getService().stream()
                .filter(service -> service.getType().equals(DidConstants.HUB_URL))
                .findFirst()
                .map(Service::getServiceEndpoint)
                .orElseThrow();
```